### PR TITLE
Update free-programming-cheatsheets.md

### PR DESCRIPTION
**Pull request by Lisaluu555**
## What does this PR do?
Add resource(s)

### Description
Terraform Cheat Sheet

### Why is this valuable (or not)?
Free

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -401,6 +401,7 @@
 ### Terraform
 
 * [Terraform Cheatsheet](https://acloudguru.com/blog/engineering/the-ultimate-terraform-cheatsheet) - Eric Pulsifer (HTML)
+* [Terraform Cheat Sheet](https://www.techbeatly.com/terraform-cheat-sheet/) - Gineesh Madapparambath (HTML)
```
Comment (shrn01) [inline more/free-programming-cheatsheets.md:404]:
```suggestion
* [Terraform Cheat Sheet](https://www.techbeatly.com/terraform-cheat-sheet) - Gineesh Madapparambath (HTML)
```

**Review by shrn01 — CHANGES_REQUESTED** [review]
Please collaborate with #8457 and #8458

File: more/free-programming-cheatsheets.md [hunk 2]
```diff
@@ -401,6 +401,7 @@
 ### Terraform
 
 * [Terraform Cheatsheet](https://acloudguru.com/blog/engineering/the-ultimate-terraform-cheatsheet) - Eric Pulsifer (HTML)
+* [Terraform Cheat Sheet](https://www.techbeatly.com/terraform-cheat-sheet) - Gineesh Madapparambath (HTML)
```
Comment (shrn01) [inline more/free-programming-cheatsheets.md:404]: alphabetically order the entries. i.e. swap lines 404 with 403

**Comment by Lisaluu555** [comment]
Solved

**Pull request merged**